### PR TITLE
Changing the way we parse group flags

### PR DESCRIPTION
**Pull request by nekiro**
Ideas taken from #2248, #1394

### CODE REVIEW

**Comment by DSpeichert** [comment]
It looks ugly. But I can't think of a better way in XML.
Maybe get rid of `<flags>`? Nothing else is a child of the "group" anyway.

**Comment by ranisalt** [comment]
Maybe we could add an `inherit` option to copy from another group defaults.

**Comment by nekiro** [comment]
@ranisalt that's a good idea, do we agree on that?

File: src/groups.cpp [hunk 1]
```diff
@@ -37,10 +78,25 @@ bool Groups::load()
 		Group group;
 		group.id = pugi::cast<uint32_t>(groupNode.attribute("id").value());
 		group.name = groupNode.attribute("name").as_string();
-		group.flags = pugi::cast<uint64_t>(groupNode.attribute("flags").value());
 		group.access = groupNode.attribute("access").as_bool();
 		group.maxDepotItems = pugi::cast<uint32_t>(groupNode.attribute("maxdepotitems").value());
 		group.maxVipEntries = pugi::cast<uint32_t>(groupNode.attribute("maxvipentries").value());
+		
+		group.flags = 0;
```
Comment (marksamman) [inline src/groups.cpp]: Remove this line and restore `group.flags = pugi::cast<uint64_t>(groupNode.attribute("flags").value());` for backwards compatibility.

**Pull request merged**